store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9675,4 +9675,16 @@ TODO: delete them permanently from here
   <defaultCOSValue>TRUE</defaultCOSValue>
   <desc>This attribute is used to enable/disable zimbra headers on email printouts</desc>
 </attr>
+
+<attr id="5008" name="zimbraUserDesignation" type="string" cardinality="single" optionalIn="account" flags="accountInfo" since="8.8.8">
```

Comment:
The attribute names and descriptions are likely too general for inclusion in the product (assuming we're going to make this part of the standard ZCS product and not a one-off fork, right?). The attribute naming should help to indicate which attributes are related. The attribute descriptions should also indicate how they are used and the relationships with other attributes.

In this case, perhaps a name like 'zimbraCustomTemplateUserDesignation' is better? Or perhaps others have ideas on a common 'prefix' for these three attributes?

Likewise the description should ideally give us a clue of intended use(s). Perhaps a description similar to these might be better?
- 'Designation to be used with custom templates.' or
- 'Designation for the user. May be used as a template variable in Custom Templates.'